tests/Controllers/UploadControllerTest.cs

```diff
@@ -161,7 +174,15 @@ public async Task UploadLithologyWithMultiCodeListPropertiesProvidedShouldSaveDa
         Assert.AreEqual(1, okResult.Value);
 
         // Assert imported values
-        var borehole = context.Boreholes.Include(b => b.BoreholeCodelists).Include(b => b.Stratigraphies).ThenInclude(s => s.Layers).ThenInclude(lay => lay.LayerCodelists).ToList().Find(b => b.OriginalName == "Seth Patel");
+        var borehole = context.Boreholes
+            .Include(b => b.BoreholeCodelists)
+            .Include(b => b.Stratigraphies).ThenInclude(s => s.Layers).ThenInclude(lay => lay.ColorCodelists)
+            .Include(b => b.Stratigraphies).ThenInclude(s => s.Layers).ThenInclude(lay => lay.DebrisCodelists)
+            .Include(b => b.Stratigraphies).ThenInclude(s => s.Layers).ThenInclude(lay => lay.GrainAngularityCodelists)
+            .Include(b => b.Stratigraphies).ThenInclude(s => s.Layers).ThenInclude(lay => lay.GrainShapeCodelists)
+            .Include(b => b.Stratigraphies).ThenInclude(s => s.Layers).ThenInclude(lay => lay.OrganicComponentCodelists)
+            .Include(b => b.Stratigraphies).ThenInclude(s => s.Layers).ThenInclude(lay => lay.Uscs3Codelists)
+            .ToList().Find(b => b.OriginalName == "Seth Patel");
```

Comment:
PP: Das Laden der Daten mit allen Includes in Methode auslagern und wiederverwenden im Test.